integration-mc1-16/src/main/kotlin/org/eln2/ChunkItems.kt

```diff
@@ -0,0 +1,69 @@
+package org.eln2
+
+import net.minecraft.item.Item
+import org.eln2.items.MultimeterItem
+import org.eln2.items.OreChunks
+
+/**
+ * Items added here are automatically registered.
+ */
+enum class ChunkItems(val items: Item) {
+    MULTIMETER(MultimeterItem()),
```

Comment:
This is not a chunk item and should be added in a new enum class called `MiscItems`.